.vscode/launch.json

```diff
@@ -8,7 +8,8 @@
       "program": "library_test.py",
       "args": ["-cf", "library_test.json"],
       "console": "integratedTerminal",
-      "justMyCode": true
+      "justMyCode": true,
+      "env": { "PYTHONPATH": "${workspaceFolder}${/}src" }
```

Comment:
Why do you need to specify the PYTHONPATH ? Should be part of the dev container already.

Can you post the error messag eyou receive without please ?